main/guides/getting-started/README.md

```diff
@@ -6,6 +6,16 @@ After installing the Agoric SDK, you can proceed to [starting a project](./start
 
 ## Quick Start
 
+### Automated Install
+
+If you're using VS Code as your code editor, install the official [Agoric Extension](https://marketplace.visualstudio.com/items?itemName=danbmky.agoric-vscode) to automate the Agoric SDK install process:
```

Comment:
I wonder if "official" is how we should describe it. That might suggest that it's supported by the Agoric engineering team, and I'm not sure we're there yet.

In particular, https://github.com/Agoric/agoric-sdk/issues/6419 seems to be outstanding and is out for a bounty rather than being in the engineering queue.

Reply:
@dckc thanks for the review. [Agoric/agoric#5156](https://github.com/Agoric/agoric-sdk/issues/5156) is already completed and should be closed I think (cc: @jeetraut).

also, [Agoric/agoric-sdk#6419](https://github.com/Agoric/agoric-sdk/issues/6419) has long been completed and the bounty has been paid too, so I guess it should be closed too (cc: @cboydstun).

I used the word "official" to describe it because there's already another extension on the VS Code marketplace (and registry) with the description "agoric sdk install" and it has some downloads already. So, to not confuse users on which to download, I thought "official" would better describe it.

<img width="378" alt="Screenshot 2023-02-16 at 10 24 02 PM" src="https://user-images.githubusercontent.com/58262528/219490039-a3f51801-a9a2-4fe7-a685-24467d1a832d.png">

and "official" also suggests that the extension is maintained, which I do, I published the version 1.0.7 of the Agoric Extension today to both the VS Code Registry and the Open VSX Registry, this recent version converts the extension into a "Workspace Extension" from a "UI Extension" allowing the extension to also be downloadable and run on both Gitpod and GitHub Codespaces platform, supporting more users! 🎉

---
Agoric Extension running on Gitpod:
<img width="1450" alt="Screenshot 2023-02-16 at 10 44 03 PM" src="https://user-images.githubusercontent.com/58262528/219493953-d53a5787-4da1-42e2-89b1-0aa53b625bab.png">

---
Agoric Extension running on GitHub Codespaces:
<img width="1453" alt="Screenshot 2023-02-16 at 10 48 38 PM" src="https://user-images.githubusercontent.com/58262528/219494720-5aa8dd08-9b74-464b-b94a-3cd2b84c17ac.png">


*Side Note:*
Agoric Extension as seen on the Open VSX Registry that allows it to be downloaded on Gitpod:
![image](https://user-images.githubusercontent.com/58262528/219492688-afb16458-c63c-4550-8ad1-eacb4a10afcd.png)


we could go for another description though.

Reply:
wow! very nice work!

Reply:
thanks! 🥳